tests/unit/kvmagent/test_securitygroup_handlers.py

```
@@ -0,0 +1,504 @@
+class _LinuxModule(Protocol):
+    get_all_ethernet_device_names: Callable[[], list[str]]
+
+
+class _MiscModule(Protocol):
+    ignoreerror: Callable[[Callable[..., object]], Callable[..., object]]
+
+from collections.abc import MutableSet
+
+collections = importlib.import_module("collections")
+if not hasattr(collections, "MutableSet"):
+    setattr(collections, "MutableSet", MutableSet)
```

Ruff (0.15.2)

[warning] Do not call `setattr` with a constant attribute value. It is not any safer than normal property access.

Replace `setattr` with assignment

(B010)